app/components/primer/markdown.rb

```diff
@@ -241,9 +241,9 @@ class Markdown < Primer::Component
     #
     #     <pre><code>var foo = "bar";</code></pre>
     #
-    #     <pre><code>Long, single-line code blocks should not wrap. They should horizontally scroll if they are too long. This line should be long enough to demonstrate this.</code></pre>
+    #     <pre tabindex="0"><code>Long, single-line code blocks should not wrap. They should horizontally scroll if they are too long. This line should be long enough to demonstrate this.</code></pre>
     #
-    #     <pre><code>var foo = "The same thing is true for code with syntax highlighting. A single line of code should horizontally scroll if it is really long.";</code></pre>
+    #     <pre tabindex="0"><code>var foo = "The same thing is true for code with syntax highlighting. A single line of code should horizontally scroll if it is really long.";</code></pre>
```

Comment:
axe was flagging this as violating the `scrollable-region-focusable` rule.

This tabindex ensures that keyboard uses can access this scrollable region and use right/left arrow keys to scroll.

https://dequeuniversity.com/rules/axe/4.1/scrollable-region-focusable